### PR TITLE
Flask-Security i18n

### LINKED ISSUE #626: Make the login form localizable
The login form uses FlaskLogin, which hard-code some english words in the templates.

The Serbian team managed to localize it with a dirty hack, but it should be possible to do it by configuration.

### PR DESCRIPTION
**Pull request by abulte**
Fix #626 

Before merging:
- wait for new `Flask-Security` release (see #626 for details)
- pin this new release into `install.pip` instead of my fork or `Flask-Security-Fork`

### CODE REVIEW

**Comment by abulte** [comment]
CI fails for now because of the link to the my fork in `install.pip`, should work when pinned to new version of `Flask-Security`.

File: udata/i18n.py [hunk 1]
```diff
@@ -58,6 +58,14 @@ def get_translations(self):
                                                          domain='wtforms')
                 translations.merge(wtforms_translations)
 
+                import flask_security
```
Comment (noirbizarre) [inline udata/i18n.py]: Why not at the top of the file with other imports ?

**Comment by davidbgk** [comment]
Fork with @opendatateam account if you have to keep the dependency.

**Comment by abulte** [comment]
@davidbgk I'm not sure what you mean, care to explain? Thanks.

**Comment by davidbgk** [comment]
@abulte sorry, it wasn't very explicit, I prefer `-e git+git@github.com:opendatateam/flask-security.git@7425cd0de7b25e852aabfadb641140e06b0506cb#egg=Flask_Security` as a dependency over `-e git+git@github.com:abulte/flask-security.git@7425cd0de7b25e852aabfadb641140e06b0506cb#egg=Flask_Security` if we have to stick to our own fork.

**Comment by abulte** [comment]
@davidbgk Ok I see 👍 We should fall back to Flask-Security main repo before merging this PR anyway.

File: requirements/install.pip [hunk 2]
```diff
@@ -20,7 +20,7 @@ Flask-Navigation==0.2.0
 Flask-OAuthlib==0.9.3
 flask-restplus==0.10.0
 Flask-Script==2.0.5
-Flask-Security-Fork==2.0.1
+Flask_Security==3.0.0
```
Comment (davidbgk) [inline requirements/install.pip]: 🎉

**Pull request merged**